### ISSUE TITLE
[Task]: Configurable auto-approval delay

### ISSUE THREAD

**Issue by abyrne-moz** [mozilla/addons#15061]
### Description

Provide a mechanism to easily change the initial auto-approval delay for new add-on listed versions (currently hardcoded to 24 hours).

Once the add-on has had a content review that delay should not longer apply as before, nothing is changing here.

### Acceptance Criteria

```[tasklist]
### Milestones/checkpoints
- [ ] The initial auto-approval listed delay can be customized without redeploying addons-server
```

### Checks

- [X] If I have identified that the work is specific to a repository, I have removed "repository:addons-server" or "repository:addons-frontend"

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/AMOENG-1107)

**Comment by ioanarusiczki** [comment]
@diox I've set in zadmin config `INITIAL_AUTO_APPROVAL_DELAY_FOR_LISTED`, it's 180 currently 

Content approving them does not seem to be required, for example I got auto-approval for any addon which won't have auto-approval disabled for some reasons

https://reviewers.addons-dev.allizom.org/en-US/reviewers/review/633183
https://reviewers.addons-dev.allizom.org/en-US/reviewers/review/633184
https://reviewers.addons-dev.allizom.org/en-US/reviewers/review/633185

same addon with content approval and without -> both scenarios auto-approved
https://reviewers.addons-dev.allizom.org/en-US/reviewers/review/633179
https://reviewers.addons-dev.allizom.org/en-US/reviewers/review/633181

**Comment by ioanarusiczki** [comment]
I tried on -dev and -stage again and the listed versions are auto-approved. Exceptions are addons flagged for HR which may have auto-approval automatically disabled , themes.

I tried invalid values in admin -> then the listed version will have the 24 hours set.
Content approving a version (when the config is not set or with invalid values) -> version would be auto-approved quickly after, so it's working as expected.

Next versions on the listed channels are quickly auto-approved -> do not depend on this config settings
